jgiven-spring-junit5/build.gradle

```diff
@@ -7,20 +7,19 @@ description = "Module for using Spring dependency injection together with JGiven
 dependencies {
     api project(':jgiven-spring')
     api project(':jgiven-junit5')
-    compileOnly "org.springframework:spring-context:$springVersion"
-    compileOnly "org.springframework:spring-test:$springVersion"
-    compileOnly 'org.junit.jupiter:junit-jupiter-api:5.9.0'
+
+    implementation platform(libs.junit.bom)
+
+    compileOnly libs.bundles.spring.compile
+    compileOnly 'org.junit.jupiter:junit-jupiter-api'
+
     testImplementation project(':jgiven-html5-report')
-    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.9.0'
-    testImplementation 'org.junit.platform:junit-platform-runner:1.9.0'
-    testImplementation "org.junit.jupiter:junit-jupiter-params:5.9.0"
-    testImplementation "org.springframework:spring-tx:$springVersion"
-    testImplementation "org.springframework:spring-context:$springVersion"
-    testImplementation "org.springframework:spring-test:$springVersion"
-    testImplementation "org.springframework:spring-jdbc:$springVersion"
-    testImplementation "org.hsqldb:hsqldb:$hsqldbVersion"
-    testImplementation "org.aspectj:aspectjrt:1.9.9.1"
-    testImplementation "org.aspectj:aspectjweaver:1.9.9.1"
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine'
+    testImplementation 'org.junit.platform:junit-platform-runner'
+    testImplementation 'org.junit.jupiter:junit-jupiter-params'
```

Comment:
Couldn't we also bundle these three dependencies? I mean, we need them at several places.

Reply:
A point of contention.
We should discuss this in person

Reply:
That's maybe still a point of discussion. In my opinion this could "purge" the code a bit.